target/linux/ramips/mt7620/base-files/etc/board.d/02_network

```diff
@@ -235,6 +235,12 @@ ramips_setup_interfaces()
 		ucidef_add_switch "switch0" \
 			"1:lan:3" "2:lan:4" "3:lan:1" "4:lan:2" "0:wan" "6@eth0"
 		;;
+	tplink,archer-c5-v4)
+		ucidef_add_switch "switch0"
+		ucidef_add_switch_attr "switch0" "enable" "0"
+		ucidef_add_switch "switch1" \
+			"0:lan:4" "1:lan:3" "2:lan:2" "3:lan:1" "4:wan" "7@eth0"
```

Comment:
Does MT7620 OpenWrt really support port 7? It seems all other devices use port 6.

Reply:
Yes, port 7 is correct.